apple-pie/styles/style.css

```diff
@@ -0,0 +1,79 @@
+html,
+body {
+    margin: 0;
+    padding: 0;
+    font-family: "Century Gothic";
+}
+
+header {
+    position: relative;
+    background-image: url(../images/apple-pie.jpg);
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 650px;
+}
+
+.header-bar {
+    text-align: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+h1 {
+    font-weight: lighter;
+    font-size: 60px;
+    color: white;
+}
+
+h2 {
+    font-weight: lighter;
+    font-size: 20px;
+    padding-left: 25px;
+    padding-bottom: 10px;
+}
+
+.history {
+    padding: 30px 0;
+    border-bottom: 3px solid rgb(209, 201, 201);
+}
+
+article {
+    font-size: 25px;
+    text-align: justify;
+    padding: 70px;
+    width: 70%;
+    margin: 0 auto;
+}
+
+.images {
+    margin: 0 auto;
+    display: flex;
+}
+
+.list {
+    padding: 40px 200px;
+    border-bottom: 3px solid rgb(209, 201, 201);
+    padding-bottom: 30px;
+}
+
+div {
+    padding-bottom: 30px;
+}
+
+li {
+    font-weight: lighter;
+    font-size: 15px;
+    padding: 0 5px;
+}
+
+ul {
+    list-style-type: '-';
+}
+
+ol {
+    list-style-type: '1)';
```

Comment:
If you pass it as `1)`, every `li` element will start with `1`. By using `ol`, you don't need to pass anything to be organised.